docs/concepts-basics/building-blocks/groups.md

```diff
@@ -1,5 +1,38 @@
 # Groups
 
-_Groups_ are made up of users who have roles. An organization can have one or more groups. Each project can be assigned one or more groups. Groups can be assigned to multiple projects. Groups are created independently of projects, and then assigned to them.
+_Groups_ are made up of users who have roles.
+An organization can have one or more groups.
+Each project can be assigned one or more groups.
+Groups can be assigned to multiple projects.
+Groups are created independently of projects, and then assigned to them.
 
-Organizations have a quota to limit the number of groups assigned to it. If you need to change the quota, please contact {{ defaults.helpstring }}.
+Organizations have a quota to limit the number of groups assigned to it.
+If you need to change the quota, please contact {{ defaults.helpstring }}.
+
+# Subgroups
+
+!!! warning
+    Subgroups and not commonly used in Lagoon, and you probably don't need them.
+    However, they can be useful in some situations.
+
+Groups can be nested to an arbitrary depth.
+Subgroups are groups which are a child of another group.
+
```

Comment:
```suggestion

Subgroups cannot be created or managed via the UI or the API; they will only be created via the legacy automated user sync tooling.
Groups can be nested to an arbitrary depth.
Subgroups are groups that are a child of another group.

```

Reply:
They can be created via the API, but not the CLI
```
Subgroups cannot be created or managed via the UI or the API
```

Reply:
This is great, I knew I wasn't the only one confused about subgroups!

Reply:
Added a note, hopefully that helps?